sprint-guide/schedules/schedules.njk

```diff
@@ -1,12 +1,13 @@
 ---js
 {
-     pagination: {
+    pagination: {
         data: "sprint_guide.schedules",
         size: 1,
         alias: "schedule",
         },
     permalink: "/sprint-guide/schedules/{{ pagination.items[0] | slug }}/",
-    layout: "layouts/sprint-guide/base"
+    layout: "layouts/sprint-guide/base",
+    title: "Schedules"
```

Comment:
I tried to get evaluate this to be the schedule title but could not make it work: `{{ sprint_guide.schedules[schedule].title }}`

This block seem to only accept strings for the title?